packages/eslint-plugin/src/rules/no-empty-interface.ts

```diff
@@ -15,10 +15,9 @@ export default util.createRule<Options, MessageIds>({
     docs: {
       description: 'Disallow the declaration of empty interfaces',
       recommended: 'error',
-      suggestion: true,
```

Comment:
This `meta.docs.suggestion property` was replaced by `meta.hasSuggestions` (more info in https://github.com/eslint/eslint/issues/14312).